src/src/WebServer/DevicesPage.cpp

```diff
@@ -972,6 +976,7 @@ void handle_devices_TaskSettingsPage(taskIndex_t taskIndex, uint8_t page)
       }
     }
     else {
+      #ifdef FEATURE_ESP_P2P
       // Show remote feed information.
       addFormSubHeader(F("Data Source"));
       uint8_t remoteUnit = Settings.TaskDeviceDataFeed[taskIndex];
```

Comment:
Ditto

Reply:
this is for display only ?

Reply:
Sure, but it can be of interest/importance to the user.